plugins/modules/ec2_vpc_peer.py

```diff
@@ -419,16 +417,16 @@ def main():
 
     if state == 'present':
         (changed, results) = create_peer_connection(client, module)
-        module.exit_json(changed=changed, peering_id=results)
     elif state == 'absent':
         if not peering_id and (not vpc_id or not peer_vpc_id):
             module.fail_json(msg='state is absent but one of the following is missing: peering_id or [vpc_id, peer_vpc_id]')
 
         remove_peer_connection(client, module)
     else:
         (changed, results) = accept_reject(state, client, module)
-        module.exit_json(changed=changed, peering_id=results)
 
+    formatted_results = camel_dict_to_snake_dict(results)
```

Comment:
```suggestion
    formatted_results = camel_dict_to_snake_dict(results)
```

The results from find_connection also include tags which will need converting from the boto3 format to a standard dictionary
```
    # Turn the boto3 result in to ansible friendly tag dictionary
    for peer in formatted_results:
        peer['tags'] = boto3_tag_list_to_ansible_dict(peer.get('tags', []))
```

Reply:
Ok. Let me know when your PR is done and I will continue my PR after a rebase and include all suggestions.

Reply:
#501 has now been merged

Reply:
Thanks!